<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UTNaoTool</class>
 <widget class="QMainWindow" name="UTNaoTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UTNaoTool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Frame</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="currentFrameSpin">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>/</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="numFrameEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Step Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="stepSpin">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="startSpin">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="endSpin">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="fastForwardButton">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/images/forward.png</normaloff>:/images/forward.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="5">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QPushButton" name="jointsButton">
        <property name="text">
         <string>Joints</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="logSelectButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Log Select</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="sensorButton">
        <property name="text">
         <string>Sensors</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="plotButton">
        <property name="text">
         <string>Plot</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="motionButton">
        <property name="text">
         <string>Motion</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="fullVisionButton">
        <property name="text">
         <string>Vision</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="cameraButton">
        <property name="text">
         <string>Camera</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="stateButton">
        <property name="text">
         <string>State</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="logButton">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="filesButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Files</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="teamButton">
        <property name="text">
         <string>Team Cfg</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="worldButton">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>World</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="walkButton">
        <property name="text">
         <string>Walk</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="logEditorButton">
        <property name="text">
         <string>Log Edit</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="5">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QRadioButton" name="runCoreRadio">
        <property name="text">
         <string>Run Core</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">memory_group</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="coreBehaviorsCheckBox">
        <property name="text">
         <string>Core Behaviors</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="localizationOnlyRadioButton">
        <property name="text">
         <string>Loc Only</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">core_group</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QRadioButton" name="visionOnlyRadioButton">
        <property name="text">
         <string>Vis Only</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">core_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="streamRadio">
        <property name="text">
         <string>Stream</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">memory_group</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="locVisRadioButton">
        <property name="text">
         <string>Loc + Vis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">core_group</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="viewLogRadio">
        <property name="text">
         <string>View Log</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">memory_group</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QCheckBox" name="logStreamCheckBox">
        <property name="text">
         <string>Log Stream</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="onDemandCheckBox">
        <property name="text">
         <string>Run On Demand</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QSlider" name="frameSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Log"/>
    <addaction name="actionOpen_Log"/>
    <addaction name="actionReOpen_Log"/>
    <addaction name="actionOpen_Recent_Log"/>
    <addaction name="actionSave_Log"/>
    <addaction name="actionReRun_Core"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect_to_Robot"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Log Control</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionPrevious_Frame"/>
    <addaction name="actionNext_Frame"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnect"/>
   <addaction name="menuControl"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOPen">
   <property name="text">
    <string>OPen</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_Log">
   <property name="text">
    <string>Open Log</string>
   </property>
  </action>
  <action name="actionNew_Log">
   <property name="text">
    <string>New Log</string>
   </property>
  </action>
  <action name="actionSave_Log">
   <property name="text">
    <string>Save Log</string>
   </property>
  </action>
  <action name="actionConnect_to_Robot">
   <property name="text">
    <string>Connect to Robot</string>
   </property>
  </action>
  <action name="actionPrevious_Frame">
   <property name="text">
    <string>Previous Frame</string>
   </property>
  </action>
  <action name="actionNext_Frame">
   <property name="text">
    <string>Next Frame</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpen_Recent_Log">
   <property name="text">
    <string>Open Recent Log</string>
   </property>
  </action>
  <action name="actionReRun_Core">
   <property name="text">
    <string>Re-Run Core</string>
   </property>
  </action>
  <action name="actionReOpen_Log">
   <property name="text">
    <string>Re-Open Log</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>frameSlider</tabstop>
  <tabstop>currentFrameSpin</tabstop>
  <tabstop>startSpin</tabstop>
  <tabstop>worldButton</tabstop>
  <tabstop>fullVisionButton</tabstop>
  <tabstop>filesButton</tabstop>
  <tabstop>motionButton</tabstop>
  <tabstop>logButton</tabstop>
  <tabstop>logSelectButton</tabstop>
  <tabstop>jointsButton</tabstop>
  <tabstop>stateButton</tabstop>
  <tabstop>cameraButton</tabstop>
  <tabstop>sensorButton</tabstop>
  <tabstop>plotButton</tabstop>
  <tabstop>teamButton</tabstop>
  <tabstop>walkButton</tabstop>
  <tabstop>logEditorButton</tabstop>
  <tabstop>viewLogRadio</tabstop>
  <tabstop>runCoreRadio</tabstop>
  <tabstop>streamRadio</tabstop>
  <tabstop>locVisRadioButton</tabstop>
  <tabstop>localizationOnlyRadioButton</tabstop>
  <tabstop>visionOnlyRadioButton</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="memory_group"/>
  <buttongroup name="core_group"/>
 </buttongroups>
</ui>
